src/main/drivers/io.c

```diff
@@ -238,19 +238,25 @@ void IOToggle(IO_t io)
 // claim IO pin, set owner and resources
 void IOInit(IO_t io, resourceOwner_e owner, uint8_t index)
 {
+    if (!io)
```

Comment:
Yes, good for now.

Eventually, serial driver should probably internally clear MODE_TX/MODE_RX when tx/rx pin is not available and handle tx/rx switch on F3.
But it is unrelated to this PR